iothub/service/src/Configurations/ConfigurationsClient.cs

```diff
@@ -130,7 +130,7 @@ public virtual async Task<Configuration> GetAsync(string configurationId, Cancel
             catch (Exception ex)
             {
                 if (Logging.IsEnabled)
-                    Logging.Error(this, $"Getting configuration threw an exception: {ex}", nameof(GetAsync));
+                    Logging.Error(this, $"Getting configuration: {configurationId} threw an exception: {ex}", nameof(GetAsync));
```

Comment:
In this case you can remove the colon before the configuration Id as the sentence continues.

```suggestion
                    Logging.Error(this, $"Getting configuration {configurationId} threw an exception: {ex}", nameof(GetAsync));
```